VDA-Reward-Contract/test/RewardContract.ts

```diff
@@ -4,40 +4,35 @@ import chaiAsPromised from "chai-as-promised";
 import { BigNumber, Wallet } from "ethers";
 
 import hre, { ethers , upgrades } from "hardhat"
-import { RewardToken, VDARewardContract } from "../typechain-types";
+import { VDARewardContract } from "../typechain-types";
+import { RewardToken } from "@verida/reward-token-contract/typechain-types";
 import EncryptionUtils from '@verida/encryption-utils'
 
+import { abi as TokenABI, bytecode as TokenByteCode } from "@verida/reward-token-contract/artifacts/contracts/RewardToken.sol/RewardToken.json";
```

Comment:
This now needs to use our ERC20, not a reward token.

Reply:
Updated